WcaOnRails/app/models/event.rb

```diff
@@ -22,6 +22,10 @@ def name_in(locale)
     I18n.t(id, scope: :events, locale: locale)
   end
 
+  def cellName
+    name
+  end
```

Comment:
Great, this will make implementing https://github.com/thewca/worldcubeassociation.org/issues/1054 even easier, because no ruby code will have to be updated =)